identity/src/main/java/com/stripe/android/identity/navigation/IdentityNavGraph.kt

```diff
@@ -185,13 +194,30 @@ internal fun IdentityNavGraph(
                     hasBack = false
                 )
             }
+            screen(IndividualDestination.ROUTE) {
+                IndividualScreen(
+                    navController = navController,
+                    isStandalone = IndividualDestination.isStandAlone(it),
+                    identityViewModel = identityViewModel
```

Comment:
have you checked if more than expected recompositions are being triggered due to passing the entire viewModel as a parameter? It's usually a bad practice to pass it down.

<img width="920" alt="image" src="https://user-images.githubusercontent.com/99293320/219500635-be17793e-a661-4442-a200-1f04ea761fdc.png">

Reply:
This is a legacy issue from the Fragments, created a separate [ticket](https://jira.corp.stripe.com/browse/IDPROD-5412) to fix this